libraries/AC_Fence/AC_PolyFence_loader.h

```diff
@@ -298,6 +318,13 @@ class AC_PolyFence_loader
     // succeeded.  Will be zero if fences are not loaded
     uint32_t _load_time_ms;
 
+    // scale_latlon_from_origin - given a latitude/longitude
+    // transforms the point to an offset-from-origin and deposits
+    // the result into pos_cm.
+    bool scale_latlon_from_origin(const Location &origin,
+                                  Vector2l point,
```

Comment:
could probably save a bit of copying by changing "Vector2l point" with "const Vector2l &point" although it's not a big deal really.

Reply:
Ack - made these a `const Vector2l &point`.